{"name": "test_json_file", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '../data/test.json'", "trace": "def test_json_file():\n        print(os.getcwd())\n>       with open('../data/test.json', 'r') as fp:\nE       FileNotFoundError: [Errno 2] No such file or directory: '../data/test.json'\n\ntest/test_file.py:10: FileNotFoundError"}, "attachments": [{"name": "stdout", "source": "463c5952-6e57-40b4-b77a-391d96023e7c-attachment.txt", "type": "text/plain"}], "start": 1618405183495, "stop": 1618405183495, "uuid": "5c19f8a8-94f1-45d8-b37f-032fea1a5a86", "historyId": "d50ca8f5466892ba3c198b58ff77ce88", "testCaseId": "052bea2877096ba85b4b1b5378880999", "fullName": "test.test_file#test_json_file", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_file"}, {"name": "host", "value": "avivatangdeMacBook-Pro.local"}, {"name": "thread", "value": "17942-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_file"}]}